__tests__/app.test.js

```diff
@@ -78,3 +78,27 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  it("should return status 200 and an array of all articles sorted by 'created_at' property in descending order", () => {
```

Comment:
Nice, but just to be picky, we expect this sort by and order to happen by default, which is really important so just adding ` by default` at the end of the description will make it that much more descriptive